product_tier_validation/data/product_state_data.xml

```diff
@@ -1,8 +1,30 @@
 <odoo noupdate="1">
+    <record id="product_state_cancel" model="product.state">
+        <field name="code">cancel</field>
+        <field name="name">Cancelled</field>
+        <field name="sequence">50</field>
+    </record>
+
+    <function name="write" model="ir.model.data">
+        <function name="search" model="ir.model.data">
+            <value eval="[('module', '=', 'product_state')]" />
+        </function>
+        <value eval="{'noupdate': False}" />
+    </function>
+
     <record id="product_state.product_state_sellable" model="product.state">
-        <field name="code">confirmed</field>
+        <field name="default" eval="False" />
     </record>
-    <record id="product_state.product_state_end" model="product.state">
-        <field name="code">cancel</field>
+
+    <record id="product_state.product_state_draft" model="product.state">
```

Comment:
same as above. People has to configure on their instances.

Reply:
Yes, it is their responsibility. On other hand we can utilize existing records to help them understand more about logic.